project-management.md

```diff
@@ -1,8 +1,12 @@
 # Project Management
 
-The OpenTelemetry community has limited bandwidth for managing changes which expand the scope of OpenTelemetry, or impact many SIGs within OpenTelemetry. There are three common scenarios which have this kind of impact.
+The OpenTelemetry community has limited bandwidth for managing changes which expand the scope of OpenTelemetry, or impact many SIGs within OpenTelemetry.
 
-* Non-trivial changes to the OpenTelemetry specification.
+These are common scenarios which have this kind of impact:
+
+* Non-trivial changes to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
+* Non-trivial changes to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions).
+  * Non-trivial being: introducing new conventions, forming a new working group around a subject, topic or technology and stabilization efforts.
```

Comment:
```suggestion
* Non-trivial changes to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
* Non-trivial changes to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions).
* Non-trivial being: introducing new conventions, forming a new working group around a subject, topic or technology and stabilization efforts.
```

Reply:
I added nested to only relate to the semconv repo, but I guess those also apply to the spec more or less. That's what you had in mind @svrnm ?

Reply:
Perhaps something on a new paragraph, like:

"Non-trivial changes include: introducing..."

It's still not very clear to me what is and isn't trivial. To me, the rule of thumb has been: does it impact all SDKs? Then it's non-trivial.

Reply:
@joaopgrassi I only suggested to remove the 2 additional spaces :D

Reply:
```suggestion
* Non-trivial changes to the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
* Non-trivial changes to the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions).
  * Non-trivial being: introducing new conventions, forming a new SIG around a subject, topic or technology and stabilization efforts.
```

Reply:
I think this PR aims to define that for the semconv, we should have a separate issue that adds a definition for the spec then.

Minor change request: we want to avoid the term "Working Group", all of those groups are SIGs.